guide/how-it-works/liquidity.md

```diff
@@ -0,0 +1,314 @@
+   mobileRetina = "/assets/images/guide/how-it-works/liquidity/liquidity-07-mobile@2x.png"
+   alt-text = "Rebalancing channels on the lightning network"
+   width = 1200
+   height = 800
+   layout = "full-width"
+%}
+
+While the example thus far has focused on routing nodes, mobile nodes also fit into this scenario. Now that our node operator Lori has maintained a well-connected routing node, she can use her node to extend liquidity to mobile nodes.
+
+{% include picture.html
+   image = "/assets/images/guide/how-it-works/liquidity/liquidity-08.png"
+   retina = "/assets/images/guide/how-it-works/liquidity/liquidity-08@2x.png"
+   mobile = "/assets/images/guide/how-it-works/liquidity/liquidity-08-mobile.png"
+   mobileRetina = "/assets/images/guide/how-it-works/liquidity/liquidity-08-mobile@2x.png"
+   alt-text = "Rebalancing channels on the lightning network"
+   width = 1200
+   height = 800
+   layout = "full-width"
+%}
+
+## Handling liquidity for wallet users
+
+The above examples were designed to help you understand liquidity on a conceptual level. Ideally, the user of a mobile [daily spending wallet]({{'/guide/daily-spending-wallet/' | relative_url}}) never has to consider any of the liquidity sourcing that happens behind the scenes. Consider how you might design a bitcoin product that doesn't require the user to understand lightning channel management.
+
+### Inbound liquidity
+
+Helping the user get inbound liquidity is probably more important than outbound liquidity for most use cases. Without inbound liquidity, the user can not receive bitcoin over lightning. This makes it very difficult for a new bitcoin user to receive their first bitcoin.
+
+If the user does not have bitcoin in their wallet, they do not need outbound liquidity. On the other hand, if the user has inbound liquidity, then they are free to receive bitcoin, and then end up with outbound liquidity from the bitcoin they received.
+
+One way to help your users receive inbound liquidity is through LSPs (Lightning Service Providers). You could run your own LSP service or integrate your product with an existing LSP.
+
+[Read the LSP section]({{'/guide/how-it-works/lightning-service-providers/#inbound-liquidity' | relative_url}}) to learn more. [Breez](https://breez.technology/) is an example of a wallet that does this.
+
+### Outbound liquidity
+
+There may be some situations where you need to help your user with outbound liquidity. An example of this would be a user who already owns bitcoin and wants to spend it using their lightning wallet.
+
+#### Automatic Channel Opens
+
+One way to accomplish this could be through an automatic channel open. If the user has bitcoin in an on-chain wallet, you could have them send bitcoin on-chain to the wallet, and then automatically use those funds to open a lightning channel with a well-connected node. An example of a wallet that does this is [Blixt](https://blixtwallet.github.io/).
+
+#### LSP Swap-In
```

Comment:
You can also use a Swap-out to get inbound liquidity.

Reply:
Yeah, so I absolutely agree that swap-out can be used for inbound liquidity. My only reservation is that I feel like it's for a different situation. My intention with this part of the page was to talk about situations where a wallet handles liquidity for the user automatically. Swap-out seems like a different situation -- as a user, my goal is to move my funds on-chain, perhaps for cold storage. That's my primary goal, but another result is that I end up with inbound liquidity after the swap-out.

In a situation where a wallet is helping the user out and managing the liquidity for them, I feel like it would be a little weird for the wallet to just initiate a swap-out and move those funs on-chain.

So yeah, agreed, I just feel like swap-outs are better for different situations.

Reply:
Good points, just thought I'd bring this up but makes sense that this section focuses on wallets handling liquidity